components/infieldbutton/package.json

```diff
@@ -53,5 +53,6 @@
 	],
 	"publishConfig": {
 		"access": "public"
-	}
+	},
+	"componentGuidelinesName": ""
```

Comment:
On the docs site, the [in-field button guidelines link](https://opensource.adobe.com/spectrum-css/infieldbutton.html) actually points to the [picker guidelines](https://spectrum.adobe.com/page/picker/). 

Do we want to replicate that link for in-field button? Currently, in-field button isn't rendering a guidelines card.